src/danger.rs

```diff
@@ -0,0 +1,6 @@
+pub use crate::key::blind;
```

Comment:
I think it makes sense to actually pull these methods into this module and call it something like internal and import them in key.rs